terms.html

```diff
@@ -5,12 +5,20 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made, by a specific <a>entity</a>, about a <a>resource</a>.
+This <a>resource</a> is called the <a>subject</a> of the <a>claim</a>.
+This digital representation usually consists of an (optional) identifier (an [IRI](https://datatracker.ietf.org/doc/html/rfc3987))
+that identifies this <a>subject</a>, and one or more attributes or properties (key-value pairs, or predicate-object graphs).
+The meaning of a claim (its semantics) is determined by its author.
+Claims can be authored by <a>issuers</a>, e.g., to construct a <a>verifiable credential</a> with, or 
+by <a>holders</a>, e.g., to include in a <a>verifiable presentation</a>.
+Different <a>issuers</a> and <a>verifiers</a> can reuse the same semantics for claims through the use of shared vocabularies and/or ontologies.
   </dd>
   <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
   <dd>
-A set of one or more <a>claims</a> made by an <a>issuer</a>. 
-The <a>claims</a> in a credential can be about different <a>subjects</a>.
+A set of one or more <a>claims</a> created by a single <a>issuer</a>,
```

Comment:
You can embed VCs in VCs, so it's not always a set of claims made by a single issuer.

Reply:
It's a matter of perspective. I agree that embedding a VC1 (issued by entity I1) into a VC2 (to be issued by entity I2) should not mean that the claims within VC1 are made by VC2. But that's not what I2 would assert. Rather it would assert something like (S1,P1,VC1), where S1 is some subject identifier, P1 represents some predicate that links S1 to VC1. VC2 is a set of claims, all of which are created by I2, and it is irrelevant that there is data in these claims that comes from elsewhere.

Reply:
```suggestion
A set of one or more <a>claims</a> put forth by a single <a>issuer</a>, which
may include one or more embedded verifiable credentials from other issuers,
```